Python/README.md

```diff
@@ -0,0 +1,25 @@
+# Rock, Paper, Scissors (Python)
+
+This is a simple Rock Paper Scissors game in Python
+
+## Installation
+
+Install Python in your editor. To install Python in Visual Studio Code, head to https://code.visualstudio.com/docs/python/python-tutorial
+
+## Compilation
+
+To compile this code for this application, open a terminal. In the terminal navigate to the rock-paper-scissors directory. Then run the command: 
+
+```
+python.py
+```
```

Comment:
So, while you _can_ compile python, this isn't doing that. You can probably just drop this section.